service_notification/src/main/kotlin/org/thechance/service_notification/data/collection/UserCollection.kt

```diff
@@ -15,5 +15,8 @@ data class UserCollection(
     @SerialName("device_tokens")
     val deviceTokens: List<String>,
     @SerialName("topics")
-    val topics: List<Int>
+    val topics: List<Int>,
+    @SerialName("notifications")
+    val notifications: List<String>
```

Comment:
Is it notification IDs or notification titles ?
@Shehab-m

Reply:
ids @mosayed01